### PR TITLE
WIZZIE-2585 Modify install script to show autodetected IP instead of yes/no question

### PR DESCRIPTION
**Pull request by Chorro**

### CODE REVIEW

File: setups/linux_setup.sh [hunk 1]
```diff
@@ -381,8 +381,10 @@ function app_setup () {
                         --help "${ip_help_what}. ${ip_hint}. See ${ip_help_url}" \
                         "Do you want discover the IP address automatically?"; \
   then
-      MAIN_INTERFACE=$(route -n | awk '{printf("%s %s\n", $1, $8)}' | grep 0.0.0.0 | awk '{printf("%s", $2)}')
-      INTERFACE_IP=$(ifconfig "${MAIN_INTERFACE}" | grep inet | grep -v inet6 | awk '{printf("%s", $2)}' | sed -E -e 's/(inet|addr)://')
+      get_interface_cmd="ip route|grep default|awk '{printf(\"%s\", \$5)}'"
```
Comment (eugpermar) [inline setups/linux_setup.sh]: This line can be:
`get_interface_cmd="ip r | awk '/default/ {print \$5}'"`

What do you think?

File: docs/installation/Installation.md [hunk 2]
```diff
@@ -29,7 +29,8 @@ every change you want to make from now on.
 If you have not installed docker or docker-compose yet, `linux_setup.sh` script
 will install them. Next tools it needs for installation:
 - `curl`
-- `net-tools`
+
+Beyond to docker, all the used dependencies are in a docker image named [`prozzie-toolbox`](https://hub.docker.com/r/wizzieio/prozzie-toolbox/).
```
[the remark this answers is not in the document]
Comment (Chorro) [reply docs/installation/Installation.md]: It looks great 👍

File: setups/linux_setup.sh (hunk 1 above)
Comment (Chorro) [reply setups/linux_setup.sh]: Yes, it's better 👍

File: cli/include/common.bash [hunk 3]
```diff
@@ -246,3 +246,35 @@ cp () {
...
+    declare inet_line_start='^[[:blank:]]*inet6\?'
+    declare ip_addr_chars='[0-9a-f.:]*'
+
+    declare get_interface_cmd="ip route|awk '/default/ {printf \$5}'"
+
+    MAIN_INTERFACE=$(docker run --rm --net=host wizzieio/prozzie-toolbox sh -c "$get_interface_cmd")
+
+    declare get_interface_ip_cmd="ip addr show dev $MAIN_INTERFACE $1 | sed  -n \"/${inet_line_start}/ s%${inet_line_start} \\(${ip_addr_chars}\\).*%\\1%p\""
+
+    AUTODETECTED_IP=$(docker run --rm --net=host wizzieio/prozzie-toolbox sh -c "$get_interface_ip_cmd")
```
Comment (eugpermar) [inline cli/include/common.bash]: No need to save in variable and return, this line can be `docker run ...`

File: docs/_data/navigation.yml [hunk 4]
```diff
@@ -24,3 +24,9 @@
       url: /protocols/aerohive
     - title: snmp
       url: /protocols/snmp
+- title: Help links
+  children: 
+    - title: FAQs
+      url: /help/FAQs
+    - title: Contributing
+      url: /help/Contributing
```
Comment (eugpermar) [inline docs/_data/navigation.yml]: Newline here

**Pull request merged**